t/ui/03-source.t

```diff
@@ -23,18 +23,24 @@ use Test::Mojo;
 use Test::Warnings;
 use OpenQA::Test::Case;
 
-SKIP: {
-    skip "breaks package build", 4;
-
-    OpenQA::Test::Case->new->init_data;
-
-    my $t = Test::Mojo->new('OpenQA::WebAPI');
-
-    my $test_name = 'isosize';
-
-    $t->get_ok("/tests/99938/modules/$test_name/steps/1/src")->status_is(200)
-      ->content_like(qr|inst\.d/.*$test_name.pm|i, "$test_name test source found")
-      ->content_like(qr/ISO_MAXSIZE/i,             "$test_name test source shown");
-}
-
-done_testing();
+my $schema  = OpenQA::Test::Case->new->init_data;
+my $t       = Test::Mojo->new('OpenQA::WebAPI');
+my $name    = 'installer_timezone';
+my $id      = 99938;
+my $src_url = "/tests/$id/modules/$name/steps/1/src";
+my $res     = $t->get_ok($src_url)->status_is(200);
```

Comment:
That's redundant, `$res` here is just `$t`.

Reply:
`Test::Mojo` uses a [fluent interface](https://mojolicious.org/perldoc/Mojo/Base#FLUENT-INTERFACES), like many other classes in Mojo.

Reply:
ok, done